set.mm

```diff
@@ -504455,6 +504518,24 @@ its graph has a given second element (that is, function value).
     GDJCKADUGLMNUJUFUHUJUFUHOCUIFZUGUIJZOUKBCPQFULCGBSCGBRTUJUFUKUHULAUICUAAUIU
     GLUBMUCUD $.
 
+  ${
+    fzo0th.1 $e |- ( ph -> M e. NN0 ) $.
+    fzo0th.2 $e |- ( ph -> N e. NN0 ) $.
+    $( Equality for a half open integer range starting at zero is the same as
+       equality of its upper bound.  (Contributed by Thierry Arnoux,
+       27-May-2025.) $)
+    fzo0th $p |- ( ph -> ( ( 0 ..^ M ) = ( 0 ..^ N ) <-> M = N ) ) $=
```

Comment:
I think we have to choose a different label here. To align it with ~fzoopth, I would  suggest  ~fzo0opth (the "th" in ~fzopth and ~fzoopth comes from "ordered pair theorem", and without "op" it makes no sense here).

Reply:
Good point, I've done the change in https://github.com/metamath/set.mm/pull/4855/commits/980eed034302056c78b3a07fdd6f0c21404cedbd